src/Utilities/CafeListing.ts

```diff
@@ -0,0 +1,45 @@
+import { Cafe } from '../Models/cafe';
+
+export type CafeModel = {
```

Comment:
I migt move this definitions to some viewModels folder/module

Reply:
[640eb18](https://github.com/Kentico/kontent-sample-app-react/pull/206/commits/640eb18c35029e4c3d0c1537e5ee2c26b42f9bfb)